tests/integration/TEC/Tickets/Commerce/Admin_Tables/Orders_TableTest.php

```diff
@@ -71,15 +74,15 @@ public function it_should_match_single_row() {
 
 		$html = str_replace( $this->orders['0']->ID, '{{order_id}}', $html );
 
-		$html = preg_replace(
-			'/<time datetime="(.*)" title="(.*)">(.*)<\/time>/',
-			'<time datetime="{{order_date}}" title="{{order_date}}">{{order_date}}</time>',
+		$html = str_replace(
+			wp_list_pluck( $this->orders, 'ID' ),
+			'{{order_id}}',
 			$html
 		);
```

Comment:
I think thats not needed anymore - should be covered a couple of lines above the replacement.